app/controllers/change_history_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ChangeHistoryController < ApplicationController
+  before_action :find_manual
+  before_action :authorize_user_for_changing_history
+
+  def index
+    @publication_logs = @manual.publication_logs
```

Comment:
These are ordered by date, meaning the latest change notes do show at the bottom (matches Whitehall behaviour). I kind of disliked it first, but then it does make sense. Perhaps ordering in descending order would match the presentation on the FE better (which is latest at the top) and be better for the user?